src/Arcus.Messaging.Abstractions.ServiceBus/MessageHandling/AzureServiceBusMessageRouter.cs

```diff
@@ -443,6 +457,8 @@ protected async Task TryServiceBusFallbackMessageAsync(
                 Logger.LogWarning("No fallback message handler processed the Azure Service Bus message '{MessageId}' in pump '{JobId}', abandoning message!", message.MessageId, messageContext.JobId);
                 await messageReceiver.AbandonMessageAsync(message);
```

Comment:
Wondering if this is correct?
Message could be either gone through a general message handler or not; no fallback is matched, but we still abandon the message instead of dead-lettering it?

Reply:
Discussed seperately, it is correct.